src/js/main.js

```diff
@@ -148,14 +148,23 @@ function closeSerial() {
     }
 }
 
-function closeHandler() {
+async function closeHandler() {
```

Comment:
Not sure if `async` / `await` is supported in the Chrome web app, and we haven't decided to ditch support for it yet.